meson.build

```diff
@@ -1,6 +1,6 @@
 project('p11-kit', 'c',
         version: '0.24.1',
-        meson_version: '>= 0.49')
+        meson_version: '>= 0.56')
```

Comment:
I suppose this is for `as_link_whole`; perhaps we could simply declare a new dependency with the `link_whole` keyword to make it work with 0.49?

IIRC we stick to 0.49 for compatibility reasons, as some distos only have that version. Maybe it's time to bump it anyway.

Reply:
RHEL 8 has 0.58, but I changed it anyway.